index.html

```diff
@@ -3,9 +3,61 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style/index.css">
     <title>Document</title>
 </head>
 <body>
-    
+    <H1>Casey's Shop</H1>
+    <Section class='top-nav'>
+        <h3>Casey Boyland</h3>
+            <nav>
+                <a href="http://index.html">Home</a>
+                <a href="http://about.html">About</a>
+                <a href="http://contact.html">Contact</a>
+                <a href="http://projects.html">Projects</a>
+            </nav>
+    </Section>
+
+    <Section class='my-name'>
+        <p>Casey Boyland</p>
+        <div class='my-name-link'>
+        <a href="http://projects.html">Projects</a>
```

Comment:
awesome job linking your projects 🔥 and displaying flex on them as well!
awesome job adding a button to link to the project page!